velox/docs/develop/testing/rowNumberFuzzer.rst

```diff
@@ -0,0 +1,53 @@
+================
+RowNumber Fuzzer
+================
+
+The RowNumberFuzzer is a testing tool that automatically generate equivalent query plans and then executes these plans
+to validate the consistency of the results. It works as follows:
+
+1. Data Generation: It starts by generating a random set of input data, also known as a vector. This data can
+   have a variety of encodings and data layouts to ensure thorough testing.
+2. Plan Generation: Generate a variety of logically equivalent query plans.
+  i. A base plan that uses ValuesNode as the source.
+  #. An alter plan that uses TableScanNode as the source.
+3. Query Execution: Executes those equivalent query plans using the generated data and asserts that the results are
+   consistent across different plans.
+  i. Execute the base plan, compare the result with the reference (DuckDB) and use it as the expected result.
```

Comment:
Why DuckDB and not Presto?

Reply:
Fixed by adding support for both DuckDbQueryRunner and PrestoQueryRunner.